server/sonar-server/src/main/java/org/sonar/server/component/index/ComponentIndexer.java

```diff
@@ -104,21 +104,31 @@ private void doIndexByProjectUuid(@Nullable String projectUuid, Size bulkSize) {
 
   @Override
   public void deleteProject(String projectUuid) {
-    BulkIndexer.delete(esClient, INDEX_TYPE_COMPONENT.getIndex(), esClient.prepareSearch(INDEX_TYPE_COMPONENT)
+    BulkIndexer.delete(esClient, INDEX_TYPE_COMPONENT, esClient.prepareSearch(INDEX_TYPE_COMPONENT)
       .setQuery(boolQuery()
         .filter(
           termQuery(ComponentIndexDefinition.FIELD_PROJECT_UUID, projectUuid))));
   }
 
+  @Override
+  public void createEsQueueForIndexing(DbSession dbSession, String projectUuid) {
+    // FIXME
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)